.github/workflows/weekly_sync_4.3.0_to_main.yaml

```diff
@@ -0,0 +1,22 @@
+name: Weekly Sync from branch '4.3.0' to 'main'
+
+# Schedule to run weekly on Friday at 3:00 PM UTC
+on:
+  schedule:
+    - cron: '0 3 * * 5'
```

Comment:
Wouldn't it better to run this on Friday morning LK time? So someone has time to merge this before the day ends?

Reply:
Updated to 10.30 IST